composer.json

```diff
@@ -7,6 +7,6 @@
   "minimum-stability": "dev",
   "require": {
     "drupal/core": "^10.3 || ^11",
-    "drupal/commerce": "^3.0"
+    "drupal/commerce": "^2.40"
```

Comment:
@leopitt could we use "^2.40 || ^3.0"? I think there are no reasons to restrict to use only version 2.x.

Reply:
Updated, thanks